app/assets/stylesheets/core/grids.scss

```diff
@@ -35,6 +35,12 @@
   }
 }
 
+@mixin gridThreeItemAlways() {
+  @include gridThreeItem();
+  display: flex;
+  flex: 33%;
```

Comment:
Instead of lines 40 and 41, I think you can simply do the following to persist having three items at all breakpoints:

```
@media screen and (max-width: $medium) {
    width: calc(33% - 20px);
	padding: 0;
}
```